src/pages/index.js

```diff
@@ -74,17 +96,53 @@ const handleProfileFormSubmit = (e) => {
   // profileDescription.textContent = profileDescriptionInput.value;
   // editProfileFormValidator.disableSubmitButton();
   const inputValues = profileEditPopup.getInputValues();
-  userInfo.setUserInfo({
-    name: inputValues.Name,
-    job: inputValues.Title,
-  });
+  // userInfo.setUserInfo({
+  //   name: inputValues.Name,
+  //   job: inputValues.Title,
+  // });
```

Comment:
You should also notify the user about the loading by changing the submit button text